src/tests/unit/tests/reports/components/report-sections/no-failed-instances-congrats.test.tsx

```diff
@@ -3,29 +3,38 @@
 import * as React from 'react';
 
 import { shallow } from 'enzyme';
+import { allInstanceOutcomeTypes } from 'reports/components/instance-outcome-type';
 import {
     NoFailedInstancesCongrats,
     NoFailedInstancesCongratsDeps,
 } from 'reports/components/report-sections/no-failed-instances-congrats';
 
-describe('NoFailedInstancesCongrats with default message', () => {
-    it('renders', () => {
-        const deps: NoFailedInstancesCongratsDeps = {
-            customCongratsMessage: null,
-        };
-        const wrapper = shallow(<NoFailedInstancesCongrats deps={deps} />);
+for (const outcomeType of allInstanceOutcomeTypes) {
```

Comment:
I'd recommend using a `describe.each` here instead of a foreach loop; the test logs end up formatted a little more informatively that way. Something like:

```typescript
describe.each(allOutcomeTypes)('NoFailedInstancesCongrats with outcomeType %s', (outcomeType) => {
    it('renders per snapshot with default message', () => { /* ... */ });
    it('renders per snapshot with custom message', () => { /* ... */ });
});
`
```

Reply:
Gotcha, changed it!